File: app/src/main/java/com/example/proye_1003/models/Cita.kt
package com.example.proye_1003.models

data class Cita(
    val idCita: Int? = null,
    val idPaciente: Int,
    val nombrePaciente: String? = null,
    val fechaHora: String,
    val tipoConsulta: String,
    val notas: String? = null,
    val estatus: String? = "A",
    val duracionMin: Int = 30,
    val observaciones: String? = null,
    val diagnostico: String? = null,
    val medicamentos: String? = null
)

File: Models/Cita.cs
public class Cita
{
    public int IdCita { get; set; }
    public int IdPaciente { get; set; }
    public string NombrePaciente { get; set; }
    public DateTime FechaHora { get; set; }
    public string TipoConsulta { get; set; } = null!;
    public string? Notas { get; set; }
    public string Estatus { get; set; } = "A";
    public int DuracionMin { get; set; } = 30;

    // 🔹 NUEVOS CAMPOS
    public string? Observaciones { get; set; }
    public string? Diagnostico { get; set; }
    public string? Medicamentos { get; set; }
}
